sdk/monitor/Azure.Monitor.Query/tests.yml

```diff
@@ -0,0 +1,9 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
```suggestion
  - template: ../../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Reply:
There are two modes in the yml. The beginning as '/' means from the root folder while ../.. is the relative path. Both should work in this case.

Reply:
Hmm, I was basing it on this error - https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1920999&view=results

Reply:
Ahh may be we do not need the '/' at all.

Reply:
Actually, I think you were right about the path not being the issue (it should be fine as-is though in the .NET repo we seem to always use the relative path syntax). 

I think the issue is that in the .NET [archetype-sdk-tests.yml](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-sdk-tests.yml), we don't have an artifacts parameter. I think we can try removing the `artifacts` from the parameters list here.